libcxx/docs/ReleaseNotes/19.rst

```diff
@@ -77,6 +77,8 @@ Improvements and New Features
 Deprecations and Removals
```

Comment:
Can you please link to the RFC in the commit message and in the release note? https://discourse.llvm.org/t/rfc-freezing-c-03-headers-in-libc

This is a large enough change that we should expect people to want to have more information about that.